bundles/scripting/javascript/pom.xml

```diff
@@ -59,6 +59,8 @@
                         </Import-Package>
                         <Export-Package>
                             org.apache.sling.scripting.javascript,
+                            org.mozilla.javascript;version=1.1.0,
+                            org.mozilla.javascript.ast;version=1.1.0,
                             org.mozilla.javascript;
```

Comment:
@welcomemandeep it this somehow old code? line without version=1.1.0 should not be there